unit_test/blas/Test_Blas.hpp

```diff
@@ -15,6 +15,7 @@
 #include "Test_Blas1_nrm1.hpp"
 #include "Test_Blas1_nrm2_squared.hpp"
 #include "Test_Blas1_nrm2.hpp"
+#include "Test_Blas1_nrm2w.hpp"
```

Comment:
Good catch, I did not realize this was missing when I re-organized the unit-tests last year.